homeassistant/components/qnap/const.py

```diff
@@ -1,6 +1,30 @@
 """The Qnap constants."""
+from __future__ import annotations
 
+ATTR_DRIVE = "Drive"
+ATTR_ENABLED = "Sensor Enabled"
+ATTR_IP = "IP Address"
+ATTR_MAC = "MAC Address"
+ATTR_MASK = "Mask"
+ATTR_MAX_SPEED = "Max Speed"
+ATTR_MEMORY_SIZE = "Memory Size"
+ATTR_MODEL = "Model"
+ATTR_PACKETS_TX = "Packets (TX)"
+ATTR_PACKETS_RX = "Packets (RX)"
+ATTR_PACKETS_ERR = "Packets (Err)"
+ATTR_SERIAL = "Serial #"
+ATTR_TYPE = "Type"
+ATTR_UPTIME = "Uptime"
+ATTR_VOLUME_SIZE = "Volume Size"
```

Comment:
I don't see any benefit of moving the ATTR constants to const.py.
Most of these are sensor specific, and should really be moved later into standalone sensors.

Reply:
In the meantime you want these back in sensor.py?

Reply:
Yes, this is unrelated to config-flow and you should remove all changes that are not related to config-flow.